deepspeech_pytorch/config.py

```diff
@@ -0,0 +1,114 @@
+from dataclasses import dataclass, field
+from typing import Any, List
+
+from omegaconf import MISSING, DictConfig
+
+from deepspeech_pytorch.enums import DistributedBackend, SpectrogramWindow, RNNType
+
+defaults = [
+    {"optim": "sgd"}
+]
+
+
+@dataclass
+class TrainingConfig:
+    no_cuda: bool = False  # Enable CPU only training
+    finetune: bool = False  # Fine-tune the model from checkpoint "continue_from"
+    seed: int = 123456  # Seed for generators
+    dist_backend: DistributedBackend = DistributedBackend.nccl  # If using distribution, the backend to be used
```

Comment:
Awesome, did you try to make a command line override error here yet? :)

Reply:
Seems to work great!!